src/app/features/StepDefinitionEditor/Services/MockParserService.js

```diff
@@ -49,10 +49,10 @@ var MockParserService = function MockParserService (
     }
 
     function parseUrl (mock, mockCallExpression) {
-        var url = _.last(mockCallExpression.callee.object.arguments).raw;
-        var urlRegex = new RegExp(url.replace(/^\//, '').replace(/\/$/, ''));
-        assert(urlRegex);
-        return urlRegex.source;
+        var rawUrl = _.last(mockCallExpression.callee.object.arguments).raw;
+        var url = rawUrl.replace(/^\//, '').replace(/\/$/, '').replace(/\\/g,'');

```

Comment:
Added comments to explain display url without escaping character from raw url of json file